[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]
rero-mef = {editable = true, path = "."}
Babel = ">=2.4.0"
Flask-BabelEx = ">=0.9.3"
ijson = ">=2.3"
invenio = {version = "~=3.1.0",extras = ["base", "postgresql", "auth", "metadata", "elasticsearch6"]}
uwsgi = ">=2.0"
pymarc = "*"
prompt-toolkit = "*"
pipenv = "*"
psutil = "*"
invenio-admin = "*"
idna = "==2.7"
lxml = "==4.2.5"
amqp = "*"
# TODO: causes datetime serialize error
celery = "<4.3.0"

[dev-packages]
rero-mef = {editable = true, path = "."}
Flask-Debugtoolbar = ">=0.10.1"
Sphinx = ">=1.5.1"
check-manifest = ">=0.35"
coverage = ">=4.4.1"
isort = ">=4.3.5,<4.3.10"
mock = ">=2.0.0"
pydocstyle = ">=2.0.0"
pytest = ">=3.3.1"
pytest-cov = ">=2.5.1"
pytest-invenio = ">=1.1.0"
pytest-mock = ">=1.6.0"
pytest-pep8 = ">=1.0.6"
pytest-random-order = ">=0.5.4"
pytest-runner = ">=3.0.0,<5"
pymarc = ">=3.1.10"
urllib3 = "<1.25,>=1.21.1"

[requires]
python_version = "3.6"

[scripts]
test = "python setup.py test"
build_sphinx = "python setup.py build_sphinx"
